t/lib/proxy-wasm-tests/hostcalls/src/filter.rs

```diff
@@ -102,6 +102,10 @@ impl Context for TestHttp {
                     Some(format!("called {} times", self.n_sync_calls + 1).as_str()),
                 );
             }
+            "define_metrics" => test_define_metrics(self, TestPhase::HTTPCallResponse),
+            "increment_counters" => test_increment_counters(self, TestPhase::HTTPCallResponse, None),
+            "toggle_gauges" => test_toggle_gauges(self, TestPhase::HTTPCallResponse, None),
+            "record_histograms" => test_record_metric(self, TestPhase::HTTPCallResponse),
```

Comment:
Keeping the context tests into the proper crate will allow this code to cleanup as we don't really need these tests to be invoked from this step in this crate.

Reply:
0313c03a69deea43975a08161dbb7ab7b657b9b5